### DISCUSSION TITLE
Different cameras depending on the player

### DISCUSSION THREAD

**Discussion by DinoWattz** [efroemling/ballistica#738] [Ideas]
Currently in online matches all players share the same camera as if playing locally;

I think it'd be cool if online players had a camera that focuses on them, others in the same device, and objectives such as the football flag, hockey puck, etc. 
(_this is sort of how it works while playing in the same device with friends and bots_)

but I also believe modders should be able to change how the camera works for each player, it would be possible to make fun minigames/gamemodes using custom cameras!

**Comment by HeyErfan** [comment]
currently, Bombsquad does not benefit from such a structure

**Comment by MatiasMGz** [comment]
I don't think this is possible for now... but i would like it to be implemented at some point.

**Comment by Loup-Garou911XD** [comment]
its been discussed in
#552 #574
